lntest/itest/assertions.go

```diff
@@ -1116,49 +1116,70 @@ func assertAmountSent(amt btcutil.Amount, sndr, rcvr *lntest.HarnessNode) func()
 	}
 }
 
-// assertEdgeDisabled checks that a given edge for a channel in a node has the Disabled
-// state. Passing a boolean for incoming allows distinction between the outgoing and
-// incoming edge for the channel specified by channel point
-func assertEdgeDisabled(t *harnessTest, node *lntest.HarnessNode, chanPoint *lnrpc.ChannelPoint,
-	incoming bool) {
+// assertEdgeDisabled checks that a given edge for a channel in a node has the
+// Disable state. Passing a boolean for incoming allows distinction between the
+// outgoing and incoming edge for the channel specified by channel point
+func assertEdgeDisabled(t *harnessTest, node *lntest.HarnessNode,
+	chanPoint *lnrpc.ChannelPoint, incoming bool) {
 	ctxb := context.Background()
 
 	err := wait.NoError(func() error {
 		req := &lnrpc.ChannelGraphRequest{
 			IncludeUnannounced: true,
 		}
-		ctxt, _ := context.WithTimeout(ctxb, defaultTimeout)
+		ctxt, cancel := context.WithTimeout(ctxb, defaultTimeout)
+		defer cancel()
 		chanGraph, err := node.DescribeGraph(ctxt, req)
 		if err != nil {
 			return fmt.Errorf("unable to query node %v's graph: %v", node, err)
 		}
 		numEdges := len(chanGraph.Edges)
 		if numEdges != 2 {
-			return fmt.Errorf("expected to find 2 edge in the graph, found %d", numEdges)
+			return fmt.Errorf("expected to find 2 edge in the graph, found %d",
+				numEdges)
 		}
-		var edge *lnrpc.ChannelEdge
+		var edge *lnrpc.ChannelEdge = chanGraph.Edges[0]
```

Comment:
Can remove the `var` statement and type definition and just use `edge := chanGraph.Edges[0]`. Otherwise the assignment is easily missed when quickly scanning the code.